### DISCUSSION TITLE
The system cannot find the file specidied

### DISCUSSION THREAD

**Discussion by Cabus1982** [ionuttbara/windows-defender-remover#31] [Q&A] [answered]
This [error](https://i.ibb.co/S7v26vr/Untitled.png) appears to press (N), and then the terminal closes without visible changes. 
Windows is newly [installed](https://i.ibb.co/gr7Djmb/Screenshot-2023-02-12-014343.png), updated and no major programs in the background. I’m doing something wrong? 
Thanks.

**Comment by ionuttbara** [comment]
no. 
Its the defender disabled after pressing N?

**Reply by Cabus1982** [reply]
No, nothing happens, the window closes after 4 seconds.

**Comment by ionuttbara** [comment, marked as answer]
ok.  its look like it was a conflicting name.. i've reuploaded the release. so,you can check now and write here , if are some problems in applying

**Comment by Cabus1982** [comment]
The version that fails is 12

**Comment by ionuttbara** [comment]
ok.. sometimes i've seen that but don't worry, it's about smartscreen process if the found or not.
After pressed N, and showing the message,is SmartScreen running?

**Comment by Cabus1982** [comment]
How can I verify it?. I’m already getting dizzy

**Comment by ionuttbara** [comment]
to check if the smart screen is running open the taskmanager and see if smartscreen.exe is running or not
go to in C:\Windows\System32 and check if smartscreen.exe exist or is renamed to smartscreen.plm like to in the command.

**Comment by Cabus1982** [comment]
The smartscreen.exe file is present on disk, but not as an active process
